.github/ISSUE_TEMPLATE/diagram_proposal.yml

```
@@ -58,3 +58,10 @@ body:
       options:
         - I will try and implement it myself.
         - This is a proposal which I'd love to see built into mermaid by the wonderful community.
+  - type: checkboxes
+    attributes:
+      label: Confirmation
+      description: Please confirm the following before submitting your issue.
+      options:
+        - label: I have searched for existing issues
+          required: true
```

```suggestion
  - type: checkboxes
    attributes:
      label: Confirmation
      description: Please confirm the following before submitting your issue.
      options:
        - label: I have searched for existing proposals
          required: true
```

Please move this to the top of the questions, so people won't waste time filling out the rest.